src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BeeWingListener.java

```diff
@@ -0,0 +1,73 @@
+package io.github.thebusybiscuit.slimefun4.implementation.listeners;
+
+import org.bukkit.HeightMap;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerMoveEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.utils.SlimefunUtils;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+
+/**
+ * This {@link Listener} is responsible for the slow falling effect given to the player
+ * when nearing the ground while using the Bee Wings.
+ *
+ * @author beSnow
+ * 
+ */
+public class BeeWingListener implements Listener {
+
+    public BeeWingListener(SlimefunPlugin plugin) {
+        plugin.getServer().getPluginManager().registerEvents(this, plugin);
+    }
+
+    @EventHandler(priority = EventPriority.LOWEST)
+    public void onApproachGround(PlayerMoveEvent e) {
+        Player player = e.getPlayer();
+        
+        if (!player.isGliding()) return;
+        if (player.isOnGround()) return;
+        
+        ItemStack chestplate = player.getInventory().getChestplate();
+        if (chestplate.getType() != Material.ELYTRA) return;
```

Comment:
This check is pretty redundant since the material check is one of the first lines in `SlimefunUtils.isItemSimilar` anyway. So this adds no real benefit to it, neither readability, efficiency nor performance.